### PR TITLE
Honey

### PR DESCRIPTION
**Pull request by RichardRahl123**
<!
honeyPRStuff
honeyPRStuff2
-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR

I added Honey, Bees in the hydroponics and a few recipes with honey. I also created honey iced tea which is a combo of honey and iced tea.
TO DO:
- [x] Replace seed packet sprite with jar of bees sprite
- [ ] Get a better Honey Iced Tea metamorphic sprite
## Why / Balance

Add more food and variety to cooking system
## How to test

Cook each recipe, grow some bees and turn honeycomb into honey. Make sure honey container is making honey and comb juices into honey.
## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
-add: Honey, Honey recipes, and Bees

### CODE REVIEW

**Comment by GreaseMonk** [comment]
Love it! 
Would personally like to be able to put honey in everything, (including a honey container) and the honey container should be able to contain other liquids as well but it's basically craftable fashion to go with it.

File: Resources/Prototypes/_NF/Entities/Objects/Consumable/Food/produce.yml [hunk 1]
```diff
@@ -261,3 +261,34 @@
...
+  components:
+  - type: FlavorProfile
+    flavors:
+      - honeycomb
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 20
+        reagents:
+        - ReagentId: Nutriment #place holder
```
[the remark this answers is not in the document]
Comment (RichardRahl123) [reply Resources/Prototypes/_NF/Entities/Objects/Consumable/Food/produce.yml:279]: I originally made the honey accessible by juicing it, similar to other plants. Should I just replace the nutriment with honey reagent?

[the remark this answers is not in the document]
Comment (Cheackraze) [reply Resources/Prototypes/_NF/Entities/Objects/Consumable/Food/produce.yml:279]: you can do a portion of it, as if you're eating the honeycomb and getting some of the reagent in you etc.

[the remark this answers is not in the document]
Comment (RichardRahl123) [reply Resources/Prototypes/_NF/Entities/Objects/Consumable/Food/produce.yml]: I changed the nutriment to honey and the juicing stays the same if you don't want some vitamins (honeycomb wax) in your solution.

**Review by whatston3 — APPROVED** [review]
Added that a jar of bees can be thrown and breaks into an angry bee (that does barely any damage and dies after 15 seconds - with any swingable weapon this isn't an issue).

Cleaned up a good few of the food entities into their proper folders, including pear muffins into Baked/misc.  Changed the name of the honey apple pie, added some oats (we now have TWO oat recipes), rewrote a good few of the descriptions, and pushed the string used for seed packet generation into the SeedPrototype.

Looks good to me, thanks for the feature.

**Review by ErhardSteinhauer — APPROVED** [review]
Works alright

**Pull request merged**